src/trello-provider.ts

```diff
@@ -0,0 +1,100 @@
+/* Copyright © 2021 Seneca Project Contributors, MIT License. */
+
+
+// TODO: namespace provider zone; needs seneca-entity feature
+
+import Trello from 'trello'
+
+type TrelloProviderOptions = {}
+
+function TrelloProvider(this: any, _options: any) {
+    const seneca: any = this
+    const ZONE_BASE = 'provider/trello/'
+    let trello: Trello
+
+    // NOTE: sys- zone prefix is reserved.
+
+    seneca
+        .message('sys:provider,provider:trello,get:info', get_info)
+        .message('role:entity,cmd:load,zone:provider,base:trello,name:card',
+            load_card)
+        .message('role:entity,cmd:save,zone:provider,base:github,name:card',
+            update_card)
+
+
+    async function get_info(this: any, _msg: any) {
+        return {
+            ok: true,
+            name: 'trello',
+            details: {
+                sdk: 'trello'
+            }
+        }
+    }
+
+    async function load_card(this: any, msg: any) {
+        let ent: any = null
+
+        let q: any = msg.q
+        let [boardId, cardId]: [string, string] = q.id.split('/')
+
+        let res = await trello.getCard(boardId, cardId)
+
+        if (res.id) {
+            res.id = boardId + '/' + cardId
```

Comment:
is cardid unique over all boards? if so, we can use it directly and we don't need to construct an id like this

Reply:
Indeed is unique. Thanks, @rjrodger. I updated the Trello provider and the entity-load test suite. 
Now, We are also testing the card.id value.